src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -189,49 +190,324 @@ public function get_sortable_columns() {
-			'per_page'    => $per_page,
+		$edit_post_url  = get_edit_post_link( $event );
+		$edit_post_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $edit_post_url ),
+			esc_html( $item->name )
+		);
+
+		$admin_views = tribe( 'tickets.admin.views' );
+		$context     = [
+			'icon_html'   => $this->get_ticket_type_icon( $item ),
+			'ticket_link' => $edit_post_link,
 		];
 
-		$args = [
-			'page'               => $current_page,
-			'per_page'           => $per_page,
-			'return_total_found' => true,
-			'order'              => 'DESC',
+		return $admin_views->template( 'all-tickets/table-column-name', $context, false );
+	}
+
+	/**
+	 * Get the column ID value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_id( $item ): string {
+		return $item->ID;
+	}
+
+	/**
+	 * Get the column event value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_event( $item ): string {
+		$event = $item->get_event();
+		if ( ! $event ) {
+			return '-';
+		}
+
+		$edit_post_url  = get_edit_post_link( $event );
+		$edit_post_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $edit_post_url ),
+			get_the_title( $event )
+		);
+
+		$orders_report_url  = sprintf(
+			'https://wp.lndo.site/wp-admin/edit.php?post_type=%s&page=tickets-orders&event_id=%d',
+			$event->post_type,
+			$event->ID
+		);
+		$orders_report_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $orders_report_url ),
+			esc_html__( 'Orders', 'event-tickets' )
+		);
+
+		$attendees_report_url  = sprintf(
+			'https://wp.lndo.site/wp-admin/edit.php?post_type=%s&page=tickets-attendees&event_id=%d',
+			$event->post_type,
+			$event->ID
+		);
+		$attendees_report_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $attendees_report_url ),
+			esc_html__( 'Attendees', 'event-tickets' )
+		);
+
+		$actions = [
+			'orders'    => $orders_report_link,
+			'attendees' => $attendees_report_link,
 		];
 
-		// Setup sorting args.
-		if ( tribe_get_request_var( 'orderby' ) ) {
-			$args['orderby'] = tribe_get_request_var( 'orderby' );
+		/**
+		 * Filters the actions for the event in the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $actions The actions for the event in the All Tickets Table.
+		 */
+		$actions = apply_filters( 'tec_tickets_all_tickets_table_event_actions', $actions, $event, $item );
+
+		return sprintf( '%1$s %2$s', $edit_post_link, $this->row_actions( $actions ) );
+	}
+
+	/**
+	 * Get the column start value.
```

Comment:
```suggestion
	 * Get the column start date value.
```
Start is a bit vague, can we be more specific?